Feat/add biswap routing

ElRodrigote opened the pull request:
# ⚠️ Waiting for SDK updates to be released ⚠️ 

## Fixes: 
* #1474 
* [SWA-42](https://linear.app/swaprdev/issue/SWA-42/add-biswap-to-bsc)

# Description

* Add BiSwap as routable platform for BNB Chain

![image](https://github.com/SwaprHQ/swapr-dapp/assets/21271189/a3977239-01e2-4e65-ac1b-7459e5f3baf8)


# How to test the changes

1) Pull this branch
2) Run the project locally
3) Go to Swapr landing page
4) Connect your wallet to BNB Chain
5) Any given swap should try to route through BiSwap as part of the BNB Chain routable platforms

NelsonGaldeman commented on package.json:
```diff
@@ -46,7 +46,7 @@
     "@swapr/core": "^0.3.19",
     "@swapr/periphery": "^0.3.22",
-    "@swapr/sdk": "1.8.2",
+    "@swapr/sdk": "SwaprHQ/swapr-sdk#feat/add-biswap-dex-as-routeable-platform",
```
Is this right?

ElRodrigote replied:
I think it's contextually right. Until we deploy the new SDK version with all the changes, this PR will remain as a Draft.

This version change is part of this draft for testing purposes (in case someone pulls the branch and wants to test around). As soon as we have the new SDK version, I'll update this commit and move it out from Draft :D

Same goes for this draft PR: https://github.com/SwaprHQ/swapr-dapp/pull/1828

NelsonGaldeman replied:
Gotcha, as long as it doesn't make it to production... feels risky though. But there is a big yellow sign on the PR so ok.

NelsonGaldeman reviewed (changes requested):
Except for the version on the package.json, LGTM.
(untested)

berteotti commented on src/constants/index.tsx:
```diff
@@ -650,6 +652,9 @@ export const RoutablePlatformKeysByNetwork = {
   [ChainId.BSC_TESTNET]: [],
 }
 
+/**
+ * @TODO check why we have 2 different UNISWAP routable platforms
+ */
```
We have 2 Uniswap routable platforms, one for V2 and one for V3 ,although I don't know if they V2 one is still being used. Good "question" Rorry.

berteotti reviewed (commented):
Code looks good to me. Is it me or the SVG looks a bit strange? Will approve after having a new SDK release.

0xVenky commented:
Looking good! Tested with a few swaps and routes through Biswap! :)

berteotti commented:
@ElRodrigote don't forget to generate the new analytics please

ElRodrigote commented:
> @ElRodrigote don't forget to generate the new analytics please

Analytics added tho I have doubts about the command output, please if you could check if those values are ok would be awesome since I don't know how to do those checks yet

Also optimized BiSwap SVG logo

@NelsonGaldeman I reverted the SKD version and cleaned up the `yarn.lock` file changes
